src/login/components/GroupManagement/NameForm.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { connect } from "react-redux";
+import { Field, reduxForm } from "redux-form";
+import Form from "reactstrap/lib/Form";
+import CustomInput from "../Inputs/CustomInput";
+import EduIDButton from "../../../components/EduIDButton";
+import { emptyValueValidation } from "../../app_utils/validation/emptyValueValidation";
+import i18n from "../../translation/InjectIntl_HOC_factory";
+
+let NameForm = (props) => {
```

Comment:
Can we change the NameForm to GroupNameForm?
I think that name should be more relevant for the function

Reply:
I have renamed this

Reply:
:+1: